Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -2552,7 +2571,7 @@ def fetch_incidents():
 
         detections = filter_incidents_by_duplicates_and_limit(incidents_res=detections,
                                                               last_run=current_fetch_info_detections,
-                                                              fetch_limit=fetch_limit, id_field='name')
+                                                              fetch_limit=INCIDENTS_PER_FETCH, id_field='name')
```

Comment:
is this intended? why?
relevant to the other places it was changed.

Reply:
Yes
This is the original issue @daryakoval handled: https://jira-hq.paloaltonetworks.local/browse/XSUP-27448